### PR TITLE
fix some bugs in dataset_traversal

### PR DESCRIPTION
**Pull request by BeyondYourself**
fix the fix read and remove some unused import;

improve the label read, it dose not need to split based on the ' \t' and remove the spaceblank at left/right before we split

### CODE REVIEW

**Comment by BeyondYourself** [comment]
@littletomatodonkey  please review this pr

File: ppocr/data/rec/dataset_traversal.py [hunk 1]
```diff
@@ -268,7 +264,7 @@ def sample_iter_reader():
                             infer_mode=True)
                     yield norm_img
             else:
-                with open(self.label_file_path, "rb") as fin:
```
Comment (littletomatodonkey) [inline ppocr/data/rec/dataset_traversal.py:271]: 这块改回 `with open(xxx, "rb")`吧

Comment (BeyondYourself) [reply ppocr/data/rec/dataset_traversal.py]: OK

Comment (BeyondYourself) [reply ppocr/data/rec/dataset_traversal.py:273]: 此处还是建议使用encoding,后续使用就避免出现忘记decode操作

**Pull request closed without merging**